loom/engine/loom2d/l2dRectangle.h

```diff
@@ -260,15 +260,18 @@ class Rectangle
     /**
      * Make a copy of this Rectangle.
      */
-    Rectangle *clone()
+    int clone(lua_State *L)
```

Comment:
Does this actually affect memory usage? I'd be interested in a before/after with just this change. I'm not that familiar with the lua bridge to say either way.